assets/typescript/RegistrationStateMachine.ts

```diff
@@ -62,6 +64,12 @@ export class RegistrationStateMachine {
         this.qrCode.hide();
         document.location.replace(this.finalizedUrl);
         break;
+      case RegistrationStateMachine.TIMEOUT:
+        this.qrCode.hide();
+        this.statusUi.showTimeoutHappened();
+        this.statusPollingService.stop();
+        this.previousStatus = RegistrationStateMachine.ERROR;
```

Comment:
What is tiqr-server?

Reply:
Tiqr-server is the role that Stepup-tiqr has (with help from https://github.com/Tiqr/tiqr-server-libphp). The tiqr-client is the iOS/Android app running on the user's phone.

Reply:
So a "tiqr-server" it is (part of) what Stepup-tiqr is, it not a separate server or process from Stepup-tiqr.